cfgov/jinja2/v1/landing-page/index.html

```diff
@@ -90,32 +42,27 @@
 {%- endblock %}
 
 {% block content_sidebar scoped -%}
-    {% if prototype %}
-        {{ related_posts.render(global_dict.related_posts_settings) }}
-        {% include 'organisms/email-signup.html' %}
-    {% else %}
-        {% for block in page.sidefoot %}
-            <div class="block block__flush-top">
-                {% if 'related_posts' in block.block_type %}
-                    {{ related_posts.render(block) }}
-                {% else %}
-                    {% if 'heading' in block.block_type %}
-                        <h3 class="o-sidebar-breakout_heading">
-                            {{ render_stream_child(block) }}
-                        </h3>
-                    {% elif 'slug' in block.block_type %}
-                        <h2 class="header-slug">
-                            <span class="header-slug_inner">
-                                {{ render_stream_child(block) }}
-                            </span>
-                        </h2>
-                    {% elif 'paragraph' in block.block_type %}
-                        {{ render_stream_child(block) }}
-                    {% else %}
+    {% for block in page.sidefoot %}
+        <div class="block block__flush-top">
+            {% if 'related_posts' in block.block_type %}
+                {{ related_posts.render(block) }}
+            {% else %}
+                {% if 'heading' in block.block_type %}
```

Comment:
@anselmbradford @kurtw Sorry, forgot to comment on this earlier.  It seemed a little out of scope and I wanted to keep this PR as tight as possible since it touches a lot of code. What do you think?